addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -431,34 +433,19 @@ const FontFamilyPickerUserValueWidget = SelectUserValueWidget.extend({
     },
 });
 
-const GPSPicker = InputUserValueWidget.extend({
-    // Explicitly not consider all InputUserValueWidget events. E.g. we actually
-    // don't want input focusout messing with the google map API. Because of
-    // this, clicking on google map autocomplete suggestion on Firefox was not
-    // working properly.
-    events: {},
+class GpsUserValue extends UserValue {
+    _gmapCacheGPSToPlace = {};
 
-    /**
-     * @constructor
-     */
-    init() {
-        this._super(...arguments);
-        this._gmapCacheGPSToPlace = {};
-
-        // The google API will be loaded inside the website iframe. Let's try
-        // not having to load it in the backend too and just using the iframe
-        // google object instead.
+    constructor() {
+        super(...arguments);
+        this._state._gmapLoaded = false;
+        this._state.gmapPlace = {};
```

Comment:
Not sure if this should be in the state, it's also implemented in the component below.

Reply:
We can only register the lib's listeners once the lib is loaded. The component's `useEffect` listens to this change - but indeed the component's `_gmapLoaded` variable was redundant & possibly wrong if the ref's el changed.